common-crafting.lic

```diff
@@ -460,4 +463,27 @@ module DRCC
     end
   end
 
+  def check_for_existing_sigil?(sigil, stock_number, quantity, bag, belt, info)
+    merged = Regexp.union($PRIMARY_SIGILS_PATTERN, $SECONDARY_SIGILS_PATTERN)
+
+    tmp_count = 0
+    more = 0
+    tmp_count = DRCI.count_items_in_container("#{sigil} sigil-scroll", bag)
```

Comment:
This method in common items strips out any word but the last, so we only get a count of "sigil"
```ruby
  def count_items_in_container(item, container)
    contents = DRC.bput("rummage /C #{item.split.last} in my #{container}", /^You rummage .*/, /That would accomplish nothing/)
    # This regexp avoids counting the quoted item name in the message, as
    # well as avoiding finding the item as a substring of other items.
    contents.scan(/ #{item}\W/).size
  end
  ```
might have to just use an edited version locally, eg rummage container, same match (You rummage) and use scan on that against the specific sigil?

Reply:
So far it works as intended. The same method is used for herb-stock and restock.